tgui/packages/tgui/interfaces/NtosNetDownloader.js

```diff
@@ -32,45 +46,49 @@ export const NtosNetDownloader = (props, context) => {
         )}
         <Section>
           <LabeledList>
-            <LabeledList.Item label="Disk usage">
-              <ProgressBar
-                value={disk_used}
-                minValue={0}
-                maxValue={disk_size}>
-                {`${disk_used} GQ / ${disk_size} GQ`}
-              </ProgressBar>
-            </LabeledList.Item>
+            {downloading && (
+              <LabeledList.Item label="Downloading">
+                <ProgressBar
+                  color="green"
+                  minValue={0}
+                  maxValue={downloadsize}
+                  value={downloadcompletion}>
+                  {`File: ${downloadname}, ${downloadpercentage}% complete`}
+                </ProgressBar>
+              </LabeledList.Item>
+            ) || (
+              <LabeledList.Item label="Disk usage">
+                <ProgressBar
+                  value={disk_used}
+                  minValue={0}
+                  maxValue={disk_size}>
+                  {`${disk_used} GQ / ${disk_size} GQ`}
+                </ProgressBar>
+              </LabeledList.Item>
+            )}
           </LabeledList>
         </Section>
-        <Section>
-          {downloadable_programs
-            .filter(program => program.access)
-            .map(program => (
+        <Flex>
```

Comment:
Replace Flex with Stack.

Wrap Tabs with `<Section fill fitted>` element.

Reply:
Thank you, I was referring to the other UI and didn't know about Stack. I'll clean things up and do something with the Downloading button.

Reply:
I did not wrap tabs assuming that your other PR will be merged